readthedocs.yaml

```diff
@@ -2,14 +2,17 @@
 
 version: 2
 
+sphinx:
+  configuration: docs/source/conf.py
+
 build:
-  os: ubuntu-22.04
+  os: ubuntu-20.04
```

Comment:
Just curious, why the downgrade? If there's something not available in later versions it might be useful to comment that in the file for when we look at it in the future :)

Reply:
Was just copying what we had in core, I can change it back